front/app/containers/Admin/messaging/AutomatedEmails/ExampleModal/ExampleFrame.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import styled from 'styled-components';
+
+// Components
+import { Box, Text } from '@citizenlab/cl2-component-library';
+import Frame from 'react-frame-component';
+
+// Hooks
+import { ICampaign } from 'services/campaigns';
+import { ICampaignExampleData } from 'api/campaign_examples/types';
+import useAppConfiguration from 'api/app_configuration/useAppConfiguration';
+
+// i18n
+import T from 'components/T';
+import { FormattedMessage } from 'utils/cl-intl';
+import messages from '../../messages';
+
+const StyledFrame = styled(Frame)`
+  border-radius: ${(props) => props.theme.borderRadius};
+  border: 1px solid ${(props) => props.theme.colors.divider};
+  width: 100%;
+  height: 450px;
+`;
+
+const ExampleFrame = ({
+  example,
+  campaign,
+}: {
+  example: ICampaignExampleData;
+  campaign?: ICampaign;
```

Comment:
```suggestion
  campaign: ICampaign;
```
I think we can be stricter on the type in ExampleModal. It doesn't make sense to render without a campaign + would avoid unnecessary checks.